challenges/palindrome-detector.js

```diff
@@ -21,3 +21,21 @@
 */
 
 // YOUR CODE HERE
+
+var isPalindrome = function (word) {
+  let arrayOfLetters = word.split("");
+  for (let i = 0; i <= arrayOfLetters.length; i++) {
+    if(arrayOfLetters[i] === arrayOfLetters[(arrayOfLetters.length - (i + 1))]) {
+      console.log(true);
```

Comment:
When we talk about functions more tomorrow & next week, we'll talk about why `console.log` is less useful here than using `return`, but it's good that you got this working!